egs/libri_2_mix/ConvTasNet/local/create_librispeech_metadata.py

```diff
@@ -0,0 +1,115 @@
+import os
+import argparse
+import soundfile as sf
+import pandas as pd
+import glob
+
+# Command line arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('--librispeech_dir', type=str, default=None,
+                    help='Directory where Librispeech is stored')
+
+
+def main(args):
+    librispeech_dir = args.librispeech_dir
+    # Librispeech metadata directory
+    librispeech_md_dir = os.path.join(librispeech_dir, 'metadata')
+    os.makedirs(librispeech_md_dir, exist_ok=True)
+    # Check if the LibriSpeech metadata already exist
+    try:
+        create_librispeech_metadata(librispeech_dir, librispeech_md_dir)
+    except FileExistsError:
+        print(f"Found metadata files in {librispeech_md_dir}, passing.")
+        return
```

Comment:
Cam also be replaced with 
```suggestion
create_librispeech_metadata(librispeech_dir, librispeech_md_dir)
```
Since create_librispeech_metadata handles already created files no ?

Reply:
I guess it's better to catch the exception so there is no error on stdout.